src/Mutant/Generator/MutationsGenerator.php

```diff
@@ -125,6 +133,17 @@ private function getMutationsFromFile(SplFileInfo $file, bool $onlyCovered): arr
 
         $traverser = new NodeTraverser();
         $mutators = $this->getMutators();
+        foreach ($mutators as $mutator) {
```

Comment:
Seems like we had (before this PR) a logic error here and now it's got worse: for each file, for each mutator - new `MutatorConfig` is created.

It means if we have 100 files, `$mutators = $this->getMutators();` will return 100 times *the same result* and 100 times `new MutatorConfig` will be created for particular mutator.

I suggest 

1. move `$mutators = $this->getMutators();` outside/before the loop `foreach ($files as $file) {`  in `generate()` method. It will allow calling this method once instead of X times, where X is the number of mutated files

2. since mutators will be outside the loop (no in `getMutationsFromFile` method), `MutatorConfig` also will be created only once for each mutator instead of X times